src/ec/packages/ec-component-footer-standardised/docs/footer-standardised.md

```diff
@@ -0,0 +1,194 @@
+# ECL Twig - EC Footer component
+
+npm package: `@ecl-twig/ec-component-footer-standardised`
+
+```shell
+npm install --save @ecl-twig/ec-component-footer-standardised
+```
+
+## Parameters
+
+- "sections" (array of objects) (default: []):
+  - "title" (optional) (string) OR ( object with Link component in property )
+  - "title_class_name" (optional) (string) (default: '')
+  - "description" (optional) (string) (default: '')
+  - "content_before" (optional) (string) (default: '')
+  - "list_class_name" (optional) (string) (default: '')
+  - "links" (optional) (array of Link components ) (default: []):
+- "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the icon
+- "extra_attributes" (optional) (array) (default: []) Extra attributes for icon
+  - "name" (string) Attribute name, eg. 'data-test'
+  - "value" (string) Attribute value, eg: 'data-test-1'
+
+## Example:
+
+<!-- prettier-ignore -->
+```twig
```

Comment:
missing the include component part here 
`{% include '@ecl-twig/ec-component-site-footer-standardised/footer-standardised.html.twig' with { `

Reply:
ok